test/jdk/java/net/httpclient/ShortResponseBody.java

```diff
@@ -703,7 +686,6 @@ public void setup() throws Exception {
 
     @AfterTest
     public void teardown() throws Exception {
-        if (sharedClient != null) sharedClient = null;
```

Comment:
should probably null `client` here?

Reply:
Not necessary. `client` is an instance field and will be GCed as soon as the framework is done with the test.

Reply:
Right we don't use the ReferenceTracker here yet. Maybe we should (but that's for later).